gordo_components/builder/build_model.py

```diff
@@ -314,13 +320,39 @@ def set_seed(self, seed: int):
         np.random.seed(seed)
         random.seed(seed)
 
+    @staticmethod
+    def build_split_dict(X: pd.DataFrame, tssplit: Type[BaseCrossValidator]) -> dict:
+        """
+        Get dictionary of cross-validation training dataset split metadata
+
+        Parameters
+        ----------
+        X: pd.DataFrame
+            The training dataset that will be split during cross-validation.
+        tssplit: Type[sklearn.model_selection.BaseCrossValidator]
```

Comment:
Sorry, I don't mean to be pedantic here, but now `tssplit` doesn't make sense, maybe just `splitter`?

Reply:
No, that makes sense :+1: